testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/beanvalidation/hibernate/validator/BootStrapValidationTestCase.java

```diff
@@ -56,6 +57,8 @@
     public static Archive<?> deploy() {
         final WebArchive war = ShrinkWrap.create(WebArchive.class, "testbootstrapvalidation.war");
         war.addPackage(BootStrapValidationTestCase.class.getPackage());
+        // Hibernate Validator needs the following runtime permission
+        war.addAsManifestResource(createPermissionsXmlAsset(new RuntimePermission("accessDeclaredMembers")), "permissions.xml");
```

Comment:
I admit I'm kind of a noob on this stuff and am unclear when this should just work and when it shouldn't. Some of these tests, like this one, are directly using Validation, so yes, that looks like direct use. Then others are doing a JNDI lookup of java:comp/Validator, which looks very similar to @Resource injection or CDI injection, all "EE" stuff described on http://docs.jboss.org/hibernate/validator/5.1/reference/en-US/html/validator-integration.html. AFAICT there's no point injecting if you're not going to call validate.

Admittedly all of these are uses from in-container test classes and not from EE components.

I noticed org.jboss.as.test.integration.beanvalidation.BeanValidationTestCase tests injection into an SLSB but doesn't actually use the validator. So I don't know if the validator would actually work there.

If there's a known issue in Hibernate Validator with a JIRA, and these perms are not expected to be needed once that's fixed, filing a WFLY JIRA to get rid of the perms later and noting it in the L60 comment makes sense. Or if we're just not sure what should work and what shouldn't, filing a WFLY JIRA to sort that and noting it in the comment for any doubtful ones also makes sense. I agree that none of these you've changed are obvious cases of "it should work without a permission".